elide-async/src/main/java/com/yahoo/elide/async/models/AsyncQuery.java

```diff
@@ -64,7 +64,6 @@
     @Embedded
     private AsyncQueryResult result;
 
-    @Exclude
     private String principalName;
```

Comment:
Lets include `@CreatePermission(expression = "Prefab.Role.None")` on principalName. We don't want it to be set by users, we will be setting it.